mobile-app/app/components/WalletTextInput.tsx

```diff
@@ -66,13 +75,32 @@ export const WalletTextInput = forwardRef<any, WalletTextInputProps>(function (p
       style={tailwind(`${containerStyle ?? 'w-full flex-col'}`)}
     >
       {title !== undefined &&
-        <ThemedSectionTitle
-          light={tailwind('text-gray-700')}
-          dark={tailwind('text-gray-200')}
-          testID={titleTestID}
-          text={title}
-          style={tailwind('text-base')}
-        />}
+      (
+        <View style={tailwind('flex flex-row justify-between items-center')}>
+          <ThemedSectionTitle
+            light={tailwind('text-gray-700')}
+            dark={tailwind('text-gray-200')}
+            testID={titleTestID}
+            text={title}
+            style={tailwind('text-base')}
+          />
+          {pasteButton?.onPasteButtonPress !== undefined && (
+            <TouchableOpacity
+              // eslint-disable-next-line
```

Comment:
why?

Reply:
to supress the handle naming error `Handler function for onPress prop key must be a camelCase name beginning with 'handle' only `